pkg/awsapi/generate/add_import.go

```diff
@@ -0,0 +1,73 @@
+//go:build ignore
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"go/format"
+	"go/token"
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	"github.com/kris-nova/logger"
+
+	"github.com/dave/dst"
+	"github.com/dave/dst/decorator"
+)
+
+func addDotImport(code []byte, packageName string) error {
```

Comment:
can we add a comment describing what this does? Specifically in the context of generating the AWS mocks

Reply:
Also, why this is needed, and where and how to apply it.

Reply:
Sure, I'll add a description.